setuptools/tests/test_easy_install.py

```diff
@@ -670,11 +670,11 @@ def test_setup_requires_override_nspkg(self, use_setup_cfg):
 
         with contexts.save_pkg_resources_state():
             with contexts.tempdir() as temp_dir:
-                foobar_1_archive = os.path.join(temp_dir, 'foo.bar-0.1.tar.gz')
-                make_nspkg_sdist(foobar_1_archive, 'foo.bar', '0.1')
+                foobar_1_archive = os.path.join(temp_dir, 'foo_bar-0.1.tar.gz')
+                make_nspkg_sdist(foobar_1_archive, 'foo_bar', '0.1')
```

Comment:
Per the comment in make_nspkg_sdist, distname should be `foo.bar`.

Reply:
Resolved in 8795306.